autogen/gradio_gui/README.md

```diff
@@ -0,0 +1,104 @@
+# AutoGen's Gradio GUI
+
+This is a GUI for AutoGen. It is written in Python and uses the [Gradio](https://gradio.app/) library.
+
+## ⭐Installation
+
+1. install AutoGen:
+
+```bash
+python -m pip install pyautogen
+```
+
+## ⭐Run
+
+1. create `OAI_CONFIG_LIST`, write your OpenAI API key or azure API key. For example:
+
+```
+[
+    {
+        "model": "gpt-3.5-turbo-16k",
+        "api_key": "----------------------------------",
+        "api_base": "https://--------.openai.azure.com",
+        "api_type": "azure",
+        "api_version": "2023-07-01-preview",
+        "deployment_id": "------"
+    }
+]
+
+```
+
+2. run following commands launch GUI:
+
+```bash
+export OAI_CONFIG_LIST='/path/to/OAI_CONFIG_LIST'
+export AUTOGEN_USE_DOCKER='False'
+export WEB_PORT=12345
+# for more environment variables options, please refer to the void terminal project
+
+python -m autogen.launch_gui
+```
+
+> Note:
+> When you run `launch_gui`, the program will automatically install additional dependencies with `pip install --user ...` if they are not installed.
```

Comment:
Would be nice to add a requirement.txt file so that users can install the dependencies by themselves.

Reply:
to install 
```
python -m pip install void-terminal>=0.0.9 https://github.com/binary-husky/gpt_academic/raw/master/docs/gradio-3.32.6-py3-none-any.whl
```

Reply:
> to install
> 
> ```
> python -m pip install void-terminal>=0.0.9 https://github.com/binary-husky/gpt_academic/raw/master/docs/gradio-3.32.6-py3-none-any.whl
> ```

Add this into the Installation section?